darwin/importer/importer.py

```diff
@@ -120,17 +121,24 @@ def build_attribute_lookup(dataset: "RemoteDataset") -> Dict[str, Any]:
     current_version=__version__,
     details=DEPRECATION_MESSAGE,
 )
-def get_remote_files(dataset: "RemoteDataset", filenames: List[str], chunk_size: int() = 100) -> Dict[str, int]:
+def get_remote_files(dataset: "RemoteDataset", filenames: List[str], chunk_size: int() = 100) -> Dict[str, Tuple[int, str]]:
     """Fetches remote files from the datasets in chunks; by default 100 filenames at a time"""
```

Comment:
Would also update this doc string to include a description of the output and why its necessary to extract the slot_name here and not downstream.